specification/app/resource-manager/Microsoft.App/preview/2022-11-01-preview/CommonDefinitions.json

```diff
@@ -614,7 +614,8 @@
           "description": "Storage type for the volume. If not provided, use EmptyDir.",
           "enum": [
             "AzureFile",
-            "EmptyDir"
+            "EmptyDir",
+            "Secret"
```

Comment:
Do we need any changed to CntainerApps.json ? i.e. 
[https://github.com/Azure/azure-rest-api-specs/blob/release-app-Microsoft.App-2022-11-01-preview/specification/app/resource-manager/Microsoft.App/preview/2022-11-01-preview/ContainerApps.json](https://github.com/Azure/azure-rest-api-specs/blob/release-app-Microsoft.App-2022-11-01-preview/specification/app/resource-manager/Microsoft.App/preview/2022-11-01-preview/ContainerApps.json)


And examples -
[https://github.com/Azure/azure-rest-api-specs/tree/release-app-Microsoft.App-2022-11-01-preview/specification/app/resource-manager/Microsoft.App/preview/2022-11-01-preview/examples](https://github.com/Azure/azure-rest-api-specs/tree/release-app-Microsoft.App-2022-11-01-preview/specification/app/resource-manager/Microsoft.App/preview/2022-11-01-preview/examples)

Reply:
no changes are needed in ContainerApps.json. 
as discussed examples are not needed right now, may add in future.